### ISSUE TITLE
Foundry 10 Testing 1 (10.275) Compatibililty error

### ISSUE THREAD

**Issue by spetzel2020** [spetzel2020/quick-encounters#93]
```
commons.js:1229 Error: You are accessing Actor5e#data which has been migrated to Actor5e#system
Deprecated since Version 10
Backwards-compatible support will be removed in Version 12
    at logCompatibilityWarning (commons.js:1218:19)
    at Actor5e._logDataFieldMigration (commons.js:5987:14)
    at Object.get (commons.js:5951:18)
    at QuickEncounter.getActorXP (QuickEncounter.js:1440:30)
    at QuickEncounter.computeTotalXP (QuickEncounter.js:1463:48)
    at QuickEncounter.renderTotalXPLine (QuickEncounter.js:1479:30)
    at QESheet.computeCombatantsForDisplay (QESheet.js:188:48)
    at QESheet.getData (QESheet.js:131:20)
    at QESheet._render (foundry.js:4807:29)
    at QESheet._render (foundry.js:5530:17)
```

**Comment by spetzel2020** [comment]
Fixed in 1.0.4i

**Comment by spetzel2020** [comment]
Released in 1.0.4
